evalml/automl/callbacks.py

```diff
@@ -15,23 +15,6 @@ def raise_error_callback(exception, traceback, automl, **kwargs):
     raise exception
 
 
-def log_and_save_error_callback(exception, traceback, automl, **kwargs):
```

Comment:
Yea I don't think we should support the "saving" callbacks anymore because the expectation is that they're able to modify the global automl search state from within the evaluation function which I don't think is a good idea during parallel evaluation. 

I think we could support something like this in the future if we return the exception/traceback from the remote worker but I don't think it's a high priority. The exception and traceback are displayed in the log already except in the "silent" callback so I don't see the value-add of storing the exception itself.

Reply:
Makes sense!